docs/cosmos-hub/validators/validator-setup.md

```diff
@@ -165,6 +165,15 @@ You should also be able to see your validator on the [Explorer](https://explorec
 To be in the validator set, you need to have more total voting power than the 100th validator.
 :::
 
+## Halting Your Validator
+
+When attempting to perform routine maintenance or planning for an upcoming coordinated
+upgrade, it can be useful to have your validator systematically and gracefully halt.
+You can achieve this by either setting the `halt-height` to the height at which
+you want your node to shutdown or by passing the `--halt-height` flag to `gaiad`.
+The node will shutdown with a zero exit code at that given height after committing
+the block.
+
```

Comment:
I think adding a line to copy paste would be nice for newcomers, to fill the parameter we could get the current height and add some X blocks to define when to stop

Reply:
Good question @sabau. This value can be set at any time really but the node would have to be restarted for it to take effect. There currently isn't a way to do this at runtime. I think there is some crossover functionality between this and the software upgrade module/process we plan to have. Essentially, to do this at runtime, it would have to be parameter somewhere.